### PR TITLE
2 implement footer static

### PR DESCRIPTION
**Pull request by paolomoz**
footer style
columns 1 and 4 plus footer bottom
missing: columns 2 and 3 and a- icons in footer bottom

Fix #<2>

Test URLs:
Before: https://main--moleculardevices--hlxsites.hlx.page/
After: https://2-implement-footer-static--moleculardevices--hlxsites.hlx.page/

### CODE REVIEW

File: blocks/footer/footer.css [hunk 1]
```diff
@@ -1,14 +1,188 @@
...
+    font-family: var(--footer-font-family);
+    font-size: 15px;
+    color: var(--footer-text-color);
+    line-height: 1.33;
 }
 
-footer .footer {
-  max-width: 1200px;
-  margin: auto;
+@media (min-width: 600px) {
```
Comment (mhaack) [inline blocks/footer/footer.css:17]: We align the media queries with the orginal site and use 768px etc.

File: blocks/footer/footer.css [hunk 2]
```diff
@@ -1,14 +1,188 @@
...
\ No newline at end of file
+@media (min-width: 900px) {
+  footer {
+    max-width: var(--main-max-width);
+    margin: auto;
+  }
+}
+
+footer a:any-link {
+    color: var(--footer-text-color);
```
Comment (mhaack) [inline blocks/footer/footer.css:31]: Ist that different than default? Only set diffs to global definition.

File: blocks/footer/footer.css [hunk 3]
```diff
@@ -1,14 +1,188 @@
...
+    text-decoration: none;
+}
+
+footer a:hover {
+    color: #0066a4;
```
Comment (mhaack) [inline blocks/footer/footer.css:36]: Use varibale

File: blocks/footer/footer.css [hunk 4]
```diff
@@ -1,14 +1,188 @@
...
+}
+
+footer h3 {
+    font-size: 16px;
```
Comment (mhaack) [inline blocks/footer/footer.css:40]: Use variable

File: blocks/footer/footer.css [hunk 5]
```diff
@@ -1,14 +1,188 @@
...
+
+footer > div > div > div:nth-child(3) {
+    background-color: #f0eff1;
+    margin-top:-16px;
+    padding-top:16px;
+}
+
+footer > div > div > div:nth-child(3) p {
+    border: none;
+    color: #5f696b;
```
Comment (mhaack) [inline blocks/footer/footer.css:84]: Use variable

File: blocks/footer/footer.css [hunk 6]
```diff
@@ -1,14 +1,188 @@
...
+    border-bottom: 2px solid var(--footer-blue-text-color);
+    color: var(--footer-blue-text-color);
+}
+
+.footer-news-events > div:not(:nth-child(-n+2)) {
+    display: none;
+}
+
+footer .icon svg {
+    color: #34393d;
```
Comment (mhaack) [inline blocks/footer/footer.css:178]: Use variable

File: blocks/footer/footer.js [hunk 7]
```diff
@@ -14,6 +35,25 @@ export default async function decorate(block) {
   const html = await resp.text();
   const footer = document.createElement('div');
   footer.innerHTML = html;
+
+  const titles = footer.querySelectorAll('.footer-news-events h3');
+  [...titles].forEach((title, i) => {
+    title.innerHTML = `<a>${title.textContent}</a>`;
```
Comment (mhaack) [inline blocks/footer/footer.js:41]: Creating an `a` without href will lead to accessibility issues I think `aria-label` should be added here.

**Review by mhaack — COMMENTED** [review]
In general the footer is too wide, compare with original site where footer has same width and is center like the content.

Also please format the CSS file we use 2 spaces. We might need to setup some project formatter.

File: blocks/footer/footer.css (hunk 1 above)
Comment (paolomoz) [reply blocks/footer/footer.css:17]: done for the desktop sizes. mobile missing and tracking in separate issue

File: blocks/footer/footer.css (hunk 2 above)
Comment (paolomoz) [reply blocks/footer/footer.css:31]: yes, it is #333 and I didn't find it in any existing variable in styles.css

File: blocks/footer/footer.css (hunk 3 above)
Comment (paolomoz) [reply blocks/footer/footer.css:36]: ok

File: blocks/footer/footer.css (hunk 4 above)
Comment (paolomoz) [reply blocks/footer/footer.css]: ok

File: blocks/footer/footer.css (hunk 5 above)
Comment (paolomoz) [reply blocks/footer/footer.css]: ok

File: blocks/footer/footer.css (hunk 6 above)
Comment (paolomoz) [reply blocks/footer/footer.css]: ok

File: blocks/footer/footer.js (hunk 7 above)
Comment (paolomoz) [reply blocks/footer/footer.js]: aria-label added

**Review by mhaack — APPROVED** [review]

**Pull request merged**